README.md

```diff
@@ -177,6 +186,12 @@ Figure 3 below outlines the use case of each page of the site.
 
 ## Known Problems 
 If you cut out an entire speech by highlighting, the speaker's name will not automatically be cut out, and vice versa.
+Version 2 of the app only works in the Google Chrome browser.
+When saving a play, all HTML in the script will be printed to the terminal. This does not cause problems, but it does take a while to scroll up to previous instructions and may look a little strange.
+In the play 'All's well that ends well', the character Parollus always has one deleted line. For some reason the translation of this line was unclear and so Folger's marked it as a missing or hidden line, and although there is nothing supposed to be said there, because of the way it is marked it is counted as a deleted line.
+The code has not been refactored and as such, there are several completely unused files and some of the logic in the Javascript functions is not clear or appropriately separated.
+
+
 
```

Comment:
It looks like your Tips and Tricks section was removed in your most recent commit, was this intentional?